packages/discovery-styles/scss/components/document-preview/_document-preview-pdf-viewer.scss

```diff
@@ -1,3 +1,61 @@
 .#{$prefix}--document-preview-pdf-viewer {
+  position: relative;
+}
+
+.#{$prefix}--document-preview-pdf-viewer--canvas {
+  transform-origin: left top 0px;
+}
+
+.#{$prefix}--document-preview-pdf-viewer--text {
   transform-origin: left top 0px;
+
+  //
+  // NOTE: import textLayer styles from ~pdfjs-dist/web/pdf_viewer.css
+  // @import "~pdfjs-dist/web/pdf_viewer" doesn't work for loading image
```

Comment:
Can you explain this? I'd rather not do this at all, since it will make maintenance difficult (and we'll need to update PDF.js pkg soon).

Reply:
If you need help with this, let me know and I can dig in as well.

Reply:
Thanks! I made this due to two reasons:
- the css file includes more styles other than ones for text layer. this is not critical
- when I tried `@import "~pdfjs-dist/web/pdf_viewer"`, `yarn run storybook` is blocked by errors like `Can't resolve './images/loading-icon.gif'`. The gif is in the `~pdfjs-dist/web/pdf_viewer/images`. I couldn't make it work. So it 's great help if you could take a look...

Reply:
Did some more investigation
- When we import the styles as scss from another module, we don't have issue because node-sass (and webpack) resolve the dependency in the module
- When we try to import the compiled css `css/index.css`, the image link is broken in the css file and we get error

What we can do would be:
- copy the images to `css/images/` folder in the build process. But I think this package is only for css and 'm not sure if it's good idea to include images or not

Reply:
I found:
- Importing the SCSS from an external module, it works
- Importing the SCSS from the storybook, it results in error

So, I'm guessing it's due to webpack configuration.

Reply:
Oh, I see. `pdf_viewer.css` doesn't have any "scoped" CSS -- it applies globally. Copying them into this parent SCSS rule takes care of that.

Also, `pdf_viewer.css` contains a lot more than just the lines you copied here.

However, I'm still concerned about this, since this makes maintenance complex -- whenever we upgrade `pdfjs-dist`, we have to remember that this code was copied and update it (if the original changed). Perhaps a better solution would be someway of copying it as a build step? I haven't fully thought through the best solution.

Reply:
OK. for the first step, I extracted the styles to a new file `_pdfjs_web_mixins.scss` and import the styles in `_document-preview-pdf-viewer.scss`.

I'll create a script to update the `_pdfjs_web_mixins.scss` later

Reply:
Added `yarn run prebuild` script to sync the `_pdfjs_web_mixins.scss` with the latest css from pdfjs-dist package.